### PR TITLE
Fix most flake8 complaints for src/ folder

### PR DESCRIPTION
**Pull request by bittner**
Theses changes address almost all `flake8` complaints in the `src/` folder.

### CODE REVIEW

**Comment by bittner** [comment]
This is really bad: The Travis CI stable image runs on Debian Trusty, which sports neither `python-wxgtk3.0` nor `python3-wxgtk4.0`. We have to install `python-wxgtk2.8`, but this doesn't seem to install the `wx` Python 2 module in a way to make Pylint recognize its existance, e.g.
```bash
************* Module src.pythonturtle
E:  7, 0: Unable to import 'wx' (import-error)
E:  8, 0: Unable to import 'wx.aui' (import-error)
E:  9, 0: Unable to import 'wx.lib.buttons' (import-error)
E: 10, 0: Unable to import 'wx.lib.scrolledpanel' (import-error)
```
This also makes clear that we need to test running PythonTurtle against several target distributions, and operating systems later.

It probably makes sense to switch to Docker-based builds, and prepare a Docker image with wxPython for each and every combination of wxPython + distro, to make things easier.

**Comment by bittner** [comment]
I [opened an issue](https://github.com/wxWidgets/Phoenix/issues/1003) in the wxPython project issue tracker. This will probably lead to a more lengthy discussion and/or implementation effort. But that's the only direction that makes sense, IMHO.

**Comment by bittner** [comment]
Let me try using the Travis Xenial-based infrastructure before merging. Hang on!

**Comment by cool-RR** [comment]
By the way, I think that Windows is the most popular os for PythonTurtle
users. I don't know whether that's relevant to what you're doing there, but
just FYI.

**Comment by bittner** [comment]
Well, it's relevant for me. I made my kids play with PythonTurtle, they have Dell notebooks running Ubuntu 18.04 LTS Bionic. They impress others (students and teachers) at school. That's what I want to happen: That teachers are impressed and that students are motivated to play with Python. Otherwise they use some other, really awkward LOGO-like language environments that favor semicolons and allow mixing UPPERcase and lowERcasE keywords. (No, thank you!)

Can you open an issue where we collect all the information for running Windows and macOS builds? I want to prepare the infrastructure for building installer packages also for Windows and macOS, but my main focus will be on GNU/Linux (.deb, .rpm, snap packages). If we manage to collect all the commands that make Windows builds (probably on [AppVeyor](https://www.appveyor.com/), not sure though) and builds for macOS happen (on either Travis CI, Circle CI, or GitLab) -- fine. But I can't and won't do trouble shooting for those non-free platforms.

You may want to look into the [feature set of AppVeyor](https://www.appveyor.com/docs/), and [open a free OSS project account](https://ci.appveyor.com/signup/free) with them. Would be nice to run everything off a single platform, but I don't see that they support builds on macOS. Maybe you find out.

File: src/pythonturtle.py [hunk 1]
```diff
@@ -2,23 +2,25 @@
...
 import wx.lib.buttons
 import wx.lib.scrolledpanel
 
-from customscrolledpanel import CustomScrolledPanel
+import homedirectory
 import shelltoprocess
-import turtlewidget
 import turtleprocess
-import multiprocessing
-import homedirectory; homedirectory.do()
```
Comment (bittner) [inline src/pythonturtle.py]: @cool-RR This is really bad style. There must be a better way to compute and deal with the "current working directory" and such. (See [related flake8 complaint](https://travis-ci.org/cool-RR/PythonTurtle/jobs/429498989#L584).)

Currently, this is unsolved. I think we have to refactor the project, though, to improve the situation.

Also, the project - [from Python's point of view](https://travis-ci.org/cool-RR/PythonTurtle/jobs/429499000#L706) - is currently called `src`. This is due to `__init__.py` being in the `src` folder. No good. This asks for being fixed.

**Comment by bittner** [comment]
Other than the above issue (that we can fix later, with a separate effort) this PR is ready to be merged.

**Comment by bittner** [comment]
You can merge this PR now, if you want. Thank you!

File: src/pythonturtle.py (hunk 1 above)
Comment (bittner) [reply src/pythonturtle.py]: Maybe we can do some collaborative programming to solve that in #106. Would be nice!

**Pull request merged**